Content.Goobstation.Shared/Voodoo/VoodooComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Goobstation.Shared.Voodoo;
+
+/// <summary>
+/// Component Used to track people with a certain name for the voodoo system
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class VoodooComponent : Component
+{
+    [DataField, AutoNetworkedField]
```

Comment:
```suggestion
    [DataField, AutoNetworkedField, ViewVariables(VVAccess.ReadWrite)]
```
This allows admins to edit the component using VV, which will allow them to use this for admin abuse, and admeming :fire: :fire: :fire: